src/app/backend/handler/apihandler.go

```diff
@@ -557,6 +558,11 @@ func CreateHTTPAPIHandler(iManager integration.IntegrationManager, cManager clie
 			To(apiHandler.handleLogs).
 			Writes(logs.LogDetails{}))
 
+	apiV1Ws.Route(
+		apiV1Ws.GET("/overview/{namespace}").
```

Comment:
Missing endpoint without namespace param causes 404 error. You need to add it and test like I said.